packages/viewer/src/modules/extensions/sections/SectionTool.ts

```diff
@@ -547,30 +540,24 @@ export class SectionTool extends Extension {
   }
 
   /**
-   * Creates a SectionBoxState from the current OBB
+   * Creates an OBB copy from the current OBB
    */
-  protected createSectionBoxState(): SectionBoxState {
-    return {
-      center: this.obb.center.clone(),
-      halfSize: this.obb.halfSize.clone(),
-      rotation: this.obb.rotation.clone()
-    }
+  protected createObbCopy(): OBB {
```

Comment:
Just a nitpick, but this should be called something like `createObbState` to be consistent with `applyObbState`